docs/connector-development/connector-builder-ui/incremental-sync.md

```diff
@@ -82,10 +82,12 @@ Then when a sync is triggered for the same connection the next day, the followin
 curl 'https://content.guardianapis.com/search?from-date=<b>2023-04-15T07:30:58Z</b>&to-date={`<now>`}'
 </pre>
 
+Except if the last record found has a date less than the time that the sync was run. In this case the sync run time will be stored in the state.
```

Comment:
```suggestion
:::info
If the last record read has a datetime earlier than the end time of the stream interval, the end time of the interval will be stored in the state. In this case, the sync start time will be stored in the state
:::
```

Reply:
I think you forgot to delete the "In this case, the sync start time will be stored in the state"

But loved your suggestion, will commit it.